search_that_hash/cracker/handler.py

```diff
@@ -0,0 +1,24 @@
+from search_that_hash.cracker.sth_mod import sth
+from search_that_hash.cracker.greppable_mod import runner as greppable
+from search_that_hash.cracker.fast_mod import runner as fast
+import logging
+import coloredlogs
+
+
+class main:

```

Comment:
Bad class name, you do not want to have multiple classes called the same thing with no meaningful name too :)

Reply:
Can we have doctypes on these so:

```
def greppable_crack(self):
    """This class does XYZ and this is how it does it
    It doesn't return but it changes the values of self.XYZ
     """
```

So we know what this does?